src/studio/README.md

```diff
@@ -94,6 +94,7 @@ Navigate to the designer backend folder. The first time running, or after any pa
 
 ```bash
 cd src/studio/src/designer/backend
+yarn run gulp-install-deps
 yarn --immutable
```

Comment:
`yarn --immutable` should be the first command. That is the equivalent to `npm install --ci` command.